src/viam/app/data_client.py

```diff
@@ -293,6 +362,15 @@ async def binary_data_by_ids(
     async def delete_tabular_data(self, organization_id: str, delete_older_than_days: int) -> int:
         """Delete tabular data older than a specified number of days.
 
+        ::
+
+            from viam.proto.app.data import Filter
+
+            my_filter = Filter(component_name="left_motor")
+            days_of_data_to_delete = 10
+            tabular_data = await data_client.delete_tabular_data(
+                "a12b3c4e-1234-1abc-ab1c-ab1c2d345abc", days_of_data_to_delete)
```

Comment:
Probably good to clarify in this sample that the `org_id` is in fact an `org_id` (`org_id="a12b..."`) as opposed to, say, a location or robot id. If one were to just quickly glance at this code snippet, it's not obvious what that value is supposed to be.